SUB3/more_options.py

```diff
@@ -39,78 +12,95 @@ def show_more_options(options):
+    pre_min_label.grid(row=3, column=0, sticky="e")
+
+    pre_min_entry = Entry(root, width=8)
+    pre_min_entry.insert(0, str(options["pre_min"]))
+    pre_min_entry.grid(row=3, column=1, sticky="w")
+
+    m1_max_label = Label(root, text="M1 Max:", bg="white", font=small_font)
+    m1_max_label.grid(row=4, column=0, sticky="e")
+
+    m1_max_entry = Entry(root, width=8)
+    m1_max_entry.insert(0, str(options["m1_max"]))
+    m1_max_entry.grid(row=4, column=1, sticky="w")
 
-    max_label = Label(root, text="Preload Max:", bg="white", font=small_font)
-    max_label.grid(row=3,column=0,sticky="e")
+    m1_min_label = Label(root, text="M1 Min:", bg="white", font=small_font)
+    m1_min_label.grid(row=5, column=0, sticky="e")
 
-    max_entry = Entry(root, width=8, fg='black')
-    max_entry.insert(0, str(options.pre_max))
-    max_entry.grid(row=3, column=1, sticky="w")
+    m1_min_entry = Entry(root, width=8)
+    m1_min_entry.insert(0, str(options["m1_min"]))
+    m1_min_entry.grid(row=5, column=1, sticky="w")
 
-    min_label = Label(root, text="Preload Min:", bg="white", font=small_font)
-    min_label.grid(row=4, column=0, sticky="e")
+    m1_thresh_label = Label(root, text="M1 Min:", bg="white", font=small_font)
+    m1_thresh_label.grid(row=6, column=0, sticky="e")
 
-    min_entry = Entry(root, width=8, fg='black')
-    min_entry.insert(0, str(options.pre_min))
-    min_entry.grid(row=4, column=1, sticky="w")
+    m1_thresh_entry = Entry(root, width=8)
+    m1_thresh_entry.insert(0, str(options["m1_thresh"]))
+    m1_thresh_entry.grid(row=6, column=1, sticky="w")
 
     sess_label = Label(root, text="Session #:", bg="white", font=small_font)
-    sess_label.grid(row=5, column=0, sticky="e")
+    sess_label.grid(row=7, column=0, sticky="e")
 
     sess_choice = IntVar(root)
-    sess_choice.set(options.sess)
+    sess_choice.set(options["sess"])
 
     sess_list = range(1, 16)
 
     def sess_command(event):
-        modified_options.sess = sess_choice.get()
+        modified_options["sess"] = sess_choice.get()
     sess_selector = OptionMenu(
         root, sess_choice, *sess_list, command=sess_command)
     sess_selector.configure(width=5, anchor="w")
-    sess_selector.grid(row=5, column=1, sticky="w")
+    sess_selector.grid(row=7, column=1, sticky="w")
 
     tor_disp = BooleanVar(root)
-    tor_disp.set(options.torque_display)
+    tor_disp.set(options["torque_display"])
 
     def tor_disp_command():
-        modified_options.torque_display = tor_disp.get()
+        modified_options["torque_display"] = tor_disp.get()
     torque_disp_checkbox = Checkbutton(root, bg='white', text='Display Torque',
                                        variable=tor_disp, onvalue=True, offvalue=False, command=tor_disp_command)
-    torque_disp_checkbox.grid(row=6,column=0,columnspan=2)
+    torque_disp_checkbox.grid(row=8,column=0,columnspan=2)
 
 
     def exit():
         root.running = False
         root.destroy()
 
     def ok():
```

Comment:
I think it would be good to rename these button handlers to match the pattern we used in other parts of the code (app.py).
on_ok  and on_exit